### ISSUE TITLE
Add switch for 1p3

### ISSUE THREAD

**Issue by Max0608** [dupondje/nrgkick-ha#11]
Hello,
is it possible to add a Switch for the 1p3 switching in the newest Nrg-Kick firmware? I would do it myself, but i have absolutely no idea where to start because i have no abilities in programming nor any skills in homeassistant (installed an instance just for the NRG-Kick Connection, i am using iobroker as homeautomation system.)
Thanks for the great solution to talk to the NRG-Kick
Greetings
Max

**Comment by SchoergPet** [comment]
YES. I would need that too. It would help a lot when switching from PV driven charging to fullspeed charging and back.
Please! If you could find the time to add that. It would be highly appreciated!
